app/api/models/errors/MtdErrors.scala

```diff
@@ -88,6 +88,32 @@ object SourceFormatError
       message = "The format of the supplied source is not valid"
     )
 
+object PaymentLotFormatError
+  extends MtdError(
+    code = "FORMAT_PAYMENT_LOT",
+    message = "The provided paymentLot is not valid."
+  )
+
+object PaymentLotItemFormatError
+  extends MtdError(
+    code = "FORMAT_PAYMENT_LOT_ITEM",
+    message = "The provided paymentLotItem is not valid."
+  )
+
+object StartDateFormatError
+  extends MtdError(
+    code = "FORMAT_START_DATE",
```

Comment:
The query parameters and error codes in the business spec are out of date, there's a comment from Sriram to update them but it hasn't been done yet. The params are supposed to be `dateFrom` and `dateTo`, and I believe the error codes are supposed to align with this naming, so `FORMAT_DATE_FROM` and `FORMAT_DATE_TO`. The message should also probably follow the format of `The provided FIELD is invalid`, e.g. `The provided dateFrom is invalid`.